pkg/svcat/service-catalog/plan.go

```diff
@@ -172,6 +173,54 @@ func (sdk *SDK) RetrievePlanByClassAndName(className, planName string, opts Scop
 	return sdk.retrieveSinglePlanByListOptions(strings.Join(ss, "/"), opts, listOpts)
 }
 
+// RetrievePlanByClassIDAndName gets a plan by its external name and class kube name combination.
+func (sdk *SDK) RetrievePlanByClassIDAndName(classKubeName, planName string, scopeOpts ScopeOptions) (Plan, error) {
+	var classRefSelector fields.Selector
+	findError := &multierror.Error{
+		ErrorFormat: func(errors []error) string {
+			return joinErrors("error:", errors, "\n  ")
+		},
+	}
+
+	if scopeOpts.Scope.Matches(ClusterScope) {
+		classRefSelector = fields.OneTermEqualSelector(FieldClusterServiceClassRef, classKubeName)
+		listOpts := metav1.ListOptions{
+			FieldSelector: fields.AndSelectors(
+				classRefSelector,
+				fields.OneTermEqualSelector(FieldExternalPlanName, planName),
+			).String(),
+		}
+
+		ss := []string{classKubeName, planName}
+		plan, err := sdk.retrieveSinglePlanByListOptions(strings.Join(ss, "/"), scopeOpts, listOpts)
```

Comment:
I'm a bit confused about this parameter here. So it's passing in the class's kuberentes name, and then the plan's external name, right? `abc123-def4546\myplan` seems off to me, no?

Reply:
so if you follow that method call, that field is only used in error messages. I could finangle things around for it to be the ExternalName if you think that is needed, although it would necessitate a call out to the Kube to fetch the class (and to figure out if its a cluster or namespaced class). Didn't seem worth the legwork to me.

Reply:
ah okay, cool I wasn't sure how it was being used further down. That's fine.